include/bl_provision.rst

```diff
@@ -0,0 +1,22 @@
+.. _doc_bl_provision:
+
+Bootloader provision page
```

Comment:
Should we try to avoid using the term `page` as this is heavily associated with the flash page concept?

`Bootloader provision data`
or simply
`Bootloader provisioning`
and then keep the name of the lib `bl_provision`

Reply:
Changed to `bootloader provisioning data`.